File: TransportesBackend/DTOs/CamionDTO.cs
namespace TransportesBackend.DTOs
{
    public class CamionDTO
    {
        public string Id { get; set; }
        public string Matricula { get; set; }
        public decimal CapacidadPeso { get; set; }
        public decimal CapacidadVolumen { get; set; }
        public bool Activo { get; set; }
    }
}

File: TransportesFrontend/src/app/vehiculos/camion/camion.dto.ts
export interface CamionDTO {
  id?: string;
  matricula: string;
  capacidadPeso: number;
  capacidadVolumen: number;
  activo: boolean;
}

export interface CreateCamionDTO {
  matricula: string;
  capacidadPeso: number;
  capacidadVolumen: number;
  activo: boolean;
}

export interface UpdateCamionDTO {
  matricula: string;
  capacidadPeso: number;
  capacidadVolumen: number;
  activo: boolean;
}
